pkg/handler/handler.go

```diff
@@ -646,8 +646,8 @@ func (h handler) unbind(w http.ResponseWriter, r *http.Request, params map[strin
 		return
 	}
 
+	userInfo, ok := r.Context().Value(UserInfoContext).(broker.UserInfo)
```

Comment:
We need to make sure it is clear that this could be empty. If the service catalog does not have the correct feature gate on we will not get this header. I think this should be fine but is a documentation on the service bundle contract.